Update run tasks for multistage and global run tasks (#561)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

1. Click **Create run task**. The run task is now available within the organization, and you can associate it with one or more workspaces.

### Global Run Tasks (Beta)

> While creating a new run task you may also configure whether the new task will be applied to all workspaces in the organization. This feature requires the **global-run-task** [entitlement][].

Reviewer: ```suggestion
When you create a new run task you may choose to apply to all workspaces in the organization. This feature requires the **global-run-task** [entitlement][].
```
